metastats-infrastructure/src/main/groovy/life/qbic/metastats/filter/JsonValidator.groovy

```diff
@@ -50,7 +51,8 @@ class JsonValidator implements SchemaValidator {
         report = jsonSchema.validate(node, true)
```

Comment:
```suggestion
        report = jsonSchema.validate(node, true)
```
My understanding from the documentation is that `isValid` always returns true and otherwise a ProcessingException is thrown.

Reply:
I don't understand your suggested change. What does your suggestions change here, to me it seems there is no change?